### ISSUE TITLE
Username barely visible in menubar when toolbar hidden

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5651]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.553 (probably been this way a while, but didn't check)
    OS Version      : Linux (or macOS dev server)
    R Version       : Any

### Steps to reproduce the problem
- login to RStudio, use default Textmate editor theme and Sky or Modern RStudio theme
- notice username at right end of toolbar

<img width="143" alt="2019-10-29_09-13-36" src="https://user-images.githubusercontent.com/10569626/67787230-71217080-fa2d-11e9-9659-a77bf25939fc.png">

- View / Hide toolbar

### Describe the problem in detail
- username is just barely visible in menubar (if you squint)

<img width="220" alt="2019-10-29_09-14-23" src="https://user-images.githubusercontent.com/10569626/67787625-22280b00-fa2e-11e9-91d0-28862102df5e.png">

<img width="221" alt="2019-10-29_09-24-11" src="https://user-images.githubusercontent.com/10569626/67787638-2522fb80-fa2e-11e9-86e6-ce807e389867.png">

- it is visible if you use Classic RStudio theme

<img width="230" alt="2019-10-29_09-26-56" src="https://user-images.githubusercontent.com/10569626/67787750-54396d00-fa2e-11e9-9b31-3a3cd8ba96b3.png">

### Describe the behavior you expected
- should always be visible, and should be at least 4.5:1 contrast in default configuration (Textmate/modern)

### Other Info
- also happens in RSP

<img width="402" alt="2019-10-29_09-42-19" src="https://user-images.githubusercontent.com/10569626/67788988-7cc26680-fa30-11e9-970f-265149f46fc1.png">
